pkg/ensurance/analyzer/analyzer.go

```diff
@@ -421,34 +434,51 @@ func (s *AnormalyAnalyzer) getEvictPods(triggered bool, action *ensuranceapi.Avo
 	return evictPods
 }
 
-func (s *AnormalyAnalyzer) disableSchedulingMerge(acsFiltered []ecache.ActionContext, avoidanceMaps map[string]*ensuranceapi.AvoidanceAction, ae *executor.AvoidanceExecutor) (enableSchedule bool) {
+func (s *AnormalyAnalyzer) disableSchedulingMerge(acsFiltered []ecache.ActionContext, avoidanceMaps map[string]*ensuranceapi.AvoidanceAction, ae *executor.AvoidanceExecutor) bool {
 	var now = time.Now()
-	enableSchedule = false
-	for _, ac := range acsFiltered {
-		if ac.Triggered {
-			enableSchedule = false
-		}
-		if ac.Restored {
+	var enableScheduling, avoidance, restore bool
```

Comment:
The logic is ok , i test some situations. But still a little confused about these 3 var, what's the difference? maybe we can add some comments.

Reply:
done~